src/screens/WalletConnect/NewNanoContractTransactionScreen.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import {
+  StyleSheet,
+  View,
+} from 'react-native';
+import { t } from 'ttag';
+import HathorHeader from '../../components/HathorHeader';
+import OfflineBar from '../../components/OfflineBar';
+import { NewNanoContractTransactionRequest } from '../../components/WalletConnect/NanoContract/NewNanoContractTransactionRequest';
+import { COLORS } from '../../styles/themes';
+
+export function NewNanoContractTransactionScreen({ route }) {
```

Comment:
Why don't you use `useRoute()`?

Reply:
This is a screen component and it is served by the navigation itself, which make it possible to get the route in its prop directly. I don't see why not get it.

Reply:
You're right, just checked their documentation and it's preferable to get it from props whenever possible